tools/clean.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+"""
+Clean the world.
+
+Note that this is probably only a partial clean - in particular, no attempt
+is made to clean the iOS/Swift world, because markh doesn't know what that
+involves.
```

Comment:
It's possible in theory to clean an xcode project from the command-line using `xcodebuild`, see e.g. [here](https://stackoverflow.com/questions/19091769/clean-ios-xcode-target-in-command-line). May be worth a followup issue for someone to poke at.